examples/triton/docker-compose-with-web.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 4g
```

Comment:
See https://www.joyent.com/blog/optimizing-docker-on-triton#ram-cpu-and-disk-resources-for-your-containers and example in https://github.com/autopilotpattern/wordpress/blob/master/docker-compose.yml#L56